custom_components/tuya_local/devices/stadlerform_roger_purifier.yaml

```diff
@@ -25,7 +25,16 @@ primary_entity:
         - dps_val: auto
           value: 100
 secondary_entities:
+  - entity: switch
```

Comment:
This should be a lock, following the pattern used for other child lock entities in other devices (using translation_key instead of explicit name and icons)